scripts/spack/configs/linux_ubuntu_24/spack.yaml

```diff
@@ -44,101 +29,114 @@ spack:
       target: [ivybridge]
       compiler: [gcc]
       providers:
-        mpi: [mpich]
         blas: [openblas]
         lapack: [openblas]
-        gl: [opengl]
-        glu: [openglu]
 
-    # Lock down which MPI we are using
-    mpi:
+    # Lock down versions of packages we depend on
+    autoconf:
       buildable: false
-    mpich:
       externals:
-      - spec: mpich@4.0%gcc
```

Comment:
It looks like some packages were removed (e.g. spec: mpich@4.0%gcc). Is this just mirroring how this is handled in serac? Perhaps verify these changes.